MediaAdminModelBundle/Migrations/MongoDB/Version20160411173754.php

```diff
@@ -0,0 +1,793 @@
+<?php
+
+namespace OpenOrchestra\MediaAdminModelBundle\Migrations\MongoDB;
+
+use AntiMattr\MongoDB\Migrations\AbstractMigration;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Doctrine\MongoDB\Database;
+use Symfony\Component\Yaml\Parser as YamlParser;
+
+/**
+ * Media migration class
+ */
+class Version20160411173754 extends AbstractMigration implements ContainerAwareInterface
+{
+    const MEDIA_TYPE_IMAGE = 'image';
+    const MEDIA_TYPE_AUDIO = 'audio';
+    const MEDIA_TYPE_DEFAULT = 'default';
+    const THUMBNAIL_IMAGE_PREFIX = 'media_thumbnail-';
+    const THUMBNAIL_AUDIO = 'orchestra-media-thumbnail-audio.png';
+    const MEDIA_PREFIX = 'media-';
+    const MEDIA_CHOICE_TYPE = 'orchestra_media';
+    const TINYMCE_TYPE = 'tinymce';
+    const ALTERNATIVES_MAPPING = array(
+        'fixed_height' => 'max_height-',
+        'fixed_width' => 'max_width-',
+        'rectangle' => 'rectangle-'
+    );
+    const ALTERNATIVES_RENAMING = array(
+        'max_height' => 'fixed_height',
+        'max_width' => 'fixed_width'
+    );
+
+    private $container;
+    protected $configuration;
+
+    /**
+     * Set the container
+     *
+     * @param ContainerInterface $container
+     */
+    public function setContainer(ContainerInterface $container = null)
```

Comment:
@alavieille: To respect Symfony\Component\DependencyInjection\ContainerAwareInterface